src/commands.rs

```diff
@@ -1307,6 +1307,15 @@ struct ObslogArgs {
     /// documented and is likely to change)
     #[clap(long, short = 'T')]
     template: Option<String>,
+    /// Show patch compared to the previous version of this change
+    ///
+    /// If the previous version has different parents, it will be temporarily
```

Comment:
Are there operations which both rebase and change the contents? Should they be tested as well?